pkgs/development/python-modules/langgraph-checkpoint-sqlite/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  langgraph-checkpoint,
+  aiosqlite,
+  pytest-asyncio,
+  pytestCheckHook,
+  langgraph-sdk,
+  poetry-core,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint-sqlite";
+  version = "1.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/langchain-ai/langgraph/blob/d9adebed846e4e822106fbde7ce33e444dcecf06/libs/checkpoint-sqlite/pyproject.toml#L12

---

Fixed.